src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -298,6 +297,13 @@ void prepare_primitive_fusing::fuse_bias(program &p) {
                 const_dep.get_output_layout().count() != static_cast<size_t>(out_features)) {
                 continue;
             }
+            auto parent_out_pshape = node->get_dependency(0).get_output_layout().get_partial_shape();
+            auto eltw_out_pshape = eltw_node.get_output_layout(0).get_partial_shape();
+            auto merged_shape = node->get_output_layout().get_partial_shape();
+            bool can_broadcast = ov::PartialShape::broadcast_merge_into(merged_shape, eltw_out_pshape, broadcast_type);
```

Comment:
It seems `eltw_out_pshape` and `merged_shape` represent the same shape of the same node (`auto& eltw_node = node->as<eltwise>()`)
Shouldn't we check it like `broadcast_merge_into(parent, const, type)`?